src/lib/Components/Bidding/Screens/__tests__/__snapshots__/ConfirmBid-tests.tsx.snap

```diff
@@ -238,6 +238,7 @@ exports[`renders properly 1`] = `
       accessibilityLabel={undefined}
       accessibilityTraits={undefined}
       accessible={true}
+      alignItems="center"
```

Comment:
Gah, this was supposed to make it into a previous PR but I think I `--amended` over it 😭. So technically _this_ PR would fix https://artsyproduct.atlassian.net/browse/PURCHASE-218, haha.